src/components/atoms/ResourceQuote.jsx

```diff
@@ -0,0 +1,16 @@
+// import React from 'react'
+// import { Box, Typography } from '@material-ui/core'
+
+
+// function ResourceQuote (props) {
+//     const { styles } = props;
+//     return(
+//         <Box components="section" id='quote' className={styles.resourcePageQuote}>
+//             <Typography variant="h5">
+//             “Rough draft thinking happens when students share their unfinished, in-progress ideas and remain open to revising those ideas.” <br/> — Rough Draft Math, Dr. Amanda Jansen
+//             </Typography>
+//         </Box>
+//     );
+// }
+
+// export default ResourceQuote;
```

Comment:
Is this file needed since the quote is in the molecule, or should the quote be moved here?